components/calendly_v2/actions/get-event/get-event.mjs

```diff
@@ -5,27 +5,28 @@ export default {
   key: "calendly_v2-get-event",
   name: "Get Event",
   description: "Gets information about an Event associated with a URI.",
-  version: "0.1.1",
+  version: "0.1.2",
   type: "action",
   props: {
     calendly_v2: {
       type: "app",
       app: "calendly_v2",
     },
-    event_uuid: {
+    event_url: {
       type: "string",
-      description: "The event's unique identifier.",
+      description: "The URL of the event to retrieve information about. If you are using the Calendly webhook, you would use *{{steps.trigger.event.payload.event}}*",
```

Comment:
- Surround references to code in `` instead of **
- Since this is 2 full sentences, make sure to include a trailing period.
- Instead of "...the Calendly webhook", I'd say something like, "...a Calendly Source in the same workflow"